dd-sdk-android/src/main/kotlin/com/datadog/android/core/configuration/Configuration.kt

```diff
@@ -496,6 +496,9 @@ internal constructor(
                     }
                 } else if (it.matches(validHostNameRegEx)) {
                     it
+                } else if (it == "localhost") {
```

Comment:
should we go with `no case` comparison here ?

Reply:
Yes that sounds like a good idea